ddtrace/bootstrap/sitecustomize.py

```diff
@@ -43,6 +43,22 @@ def update_patched_modules():
         EXTRA_PATCHED_MODULES.update({module: should_patch.lower() == 'true'})
 
 
+def add_global_tags(tracer):
+    tags = {}
+    for tag in os.environ.get('DD_TRACE_TAGS', '').split(','):
+        if len(tag.split(':')) != 2:
+            log.debug("skipping malformed tracer tag")
```

Comment:
```suggestion
            log.debug('skipping malformed tracer tag')
```

inspired by @brettlangdon 😉